examples/exec_pod/main.c

```diff
@@ -0,0 +1,85 @@
+#include <kube_config.h>
+#include <malloc.h>
+#include <stdio.h>
+#include <kube_exec.h>
+
+/* 
+ * An example of call back function:
+ */
+void my_exec_data_callback(void **p_data_received, long *p_data_received_len)
+{
+    printf("%s: Received %ld bytes:\n%s", __func__, *p_data_received_len, (char *) (*p_data_received));
+}
+
+int main(int argc, char *argv[])
+{
+    char *base_path = NULL;
+    sslConfig_t *ssl_config = NULL;
+    list_t *api_keys = NULL;
+    int rc = load_kube_config(&base_path, &ssl_config, &api_keys, NULL);    /* NULL means loading configuration from $HOME/.kube/config */
```

Comment:
Did you test with load_incluster_config?

Reply:
This PR cannot support running in cluster now.
We may support it in future. ^-^

Reply:
I'm struggling with not establishing ssl connection while using load_incluster_config.
Do you know what kind of missing implementation will it to fail?

Reply:
"ssl_config" returning from `load_incluster_config` only has CA file but does not have the client certificate and private key (which are not provided in cluster) so it cannot pass authentication by Kubernetes API server. 

"api_keys" holding the authentication credential should be passed to the websocket client. And the websocket client needs updating to use this credential.

Reply:
Should I pass the token using lws_add_http_header_by_token() function?